### PR TITLE
Add Stockholm docs

### LINKED ISSUE #1853: Add Stockholm
[Stockholm.docx](https://github.com/slurpcode/slurp/files/9687954/Stockholm.docx)

https://slurp.readthedocs.io/en/latest/

### PR DESCRIPTION
**Pull request by Kevan-Y**
Fixes #1853

 ## Description 
 - Add Stockholm docs

### CODE REVIEW

File: doc/source/stockholm.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Sodermalm neighborhood
+
+Södermalm is located on an island of the same name and is a bohemian neighborhood with a great atmosphere, art galleries, alternative shops and designer cafes. But it also has other points of interest such as the church of Santa Catalina, the Museum of Photography, the Church of María Magdalena, the Medborgarplatsen Square and Götagatan street where you can find a wide variety of stalls with vintage clothing. It can also bring you closer to the Mariatorget park at sunset or go up the Katarina elevator, where you can have incredible views of the city while the beautiful sunset occurs.
+
+## Sergel Square
+
+This original square has the Kulturhuset, which is a cultural center full of galleries, bars and interesting activities at all hours, as well as a fountain and a spectacular glass obelisk.
+Stockholm is an especially beautiful city full of interesting things to do and places to see.
```
Comment (jbampton) [inline doc/source/stockholm.md:77]:
```suggestion
Stockholm is an especially beautiful city full of interesting things to do and places to see.

## About the Author

Idais, Graduated in Mechanical Engineering, and a master’s degree in teaching component, she gave classes in several institutes of mathematics and physics, but she also dedicated several years of my life as a television producer, she did the scripts for mikes, the camera direction, editing of video and even the location. Later she was dedicated to SEO writing for a couple of years. She likes poetry, chess and dominoes.
```

File: doc/source/stockholm.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Skansen-Museum
+
+This open-air museum located on the island of Djurgarden was opened in 1891 and is considered to be the oldest in the world. In this room with a large green space, the culture and history of Sweden in the time before the industrial revolution is shown. You'll be able to see more than 150 characteristic houses and farms from that era for each region of the country, including the town hall, an 18th-century stave church, and the Bollnästorget market square.
+
+## Drottningholm Palace
+
+Drottningholm Palace was declared a UNESCO World Heritage Site. The palace built at the end of the 16th century is located on Lovön Island and is the best representation of the Swedish imperial era. Many tourists equate its beauty with the Palace of Versailles and during your visit you can enjoy the Royal Theater, the exquisite decoration of the rooms, the design of the Chinese Pavilion and its beautiful gardens.
+
+![drottningholm_palace](_static/images/stockholm/drottningholm_palace.png)
```
Comment (jbampton) [inline doc/source/stockholm.md:60]:
```suggestion
![Drottningholm Palace](_static/images/stockholm/drottningholm_palace.png)
```

File: doc/source/stockholm.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Kungstradgarden Park
+
+The Kungstradgarden Park offers you a beautiful walk among plants, fountains, trees and flowers. But it also has several cafes where you can drink something hot and if it's winter, you can skate on its ice rink.
+
+## Subway
+
+The Stockholm metro is one of the most impressive in the world, since in 90 of its 100 stations you can see them decorated with works of art by renowned artists. It is the largest art gallery in the world and although all the stations have something interesting, we are going to recommend the ones with the most important works: Radhuset, Solna Centrum, Kungsttadgarden, Tekniska Höskolan, Thorildsplan, Stadium and Slussen.
+
+![subway](_static/images/stockholm/subway.png)
```
Comment (jbampton) [inline doc/source/stockholm.md:46]:
```suggestion
![Subway](_static/images/stockholm/subway.png)
```

File: doc/source/stockholm.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# 🇸🇪 Stockholm
+
+Stockholm is the capital of Sweden and is made up of 14 islands, it is one of the most beautiful and surprising cities in Europe. Located in an archipelago with more than 24,000 islands bathed by the North Seas, this Baltic capital is known as the Venice of the North for its large number of canals.
+Stockholm is a cosmopolitan city that falls in love at first sight, with a historic center full of charming corners, it has fantastic museums and perfectly maintained parks.
+
+![stockholm](_static/images/stockholm/stockholm.png)
```
Comment (jbampton) [inline doc/source/stockholm.md:6]:
```suggestion
![Stockholm](_static/images/stockholm/stockholm.png)
```

File: doc/source/stockholm.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+We are going to guide you through this clean and tidy city, through its most iconic places for you to visit and not miss the most important things in Stockholm.
+
+## Gamla Stan
+
+Gamla Stan is the historic center of the city, with medieval buildings with colorful facades and squares such as Köpmantorget, Järntorget and Stortorget that make this part of the city a charming place. This neighborhood also has other tourist sites such as the Cathedral of Saint Nicholas, the Royal Palace, the Parliament and the Nobel Museum where you can discover the history of Alfred Nobel.
+
+![gamla_stan](_static/images/stockholm/gamla_stan.png)
```
Comment (jbampton) [inline doc/source/stockholm.md:14]:
```suggestion
![Gamla Stan](_static/images/stockholm/gamla_stan.png)
```

Comment (Kevan-Y) [reply doc/source/stockholm.md]: https://github.com/slurpcode/slurp/pull/1855/commits/35b083d23969b85d0b9a015ba66ad629a6a4d13a

File: doc/source/stockholm.md (hunk 4 above)
Comment (Kevan-Y) [reply doc/source/stockholm.md]: https://github.com/slurpcode/slurp/pull/1855/commits/35b083d23969b85d0b9a015ba66ad629a6a4d13a

File: doc/source/stockholm.md (hunk 3 above)
Comment (Kevan-Y) [reply doc/source/stockholm.md]: https://github.com/slurpcode/slurp/pull/1855/commits/35b083d23969b85d0b9a015ba66ad629a6a4d13a

File: doc/source/stockholm.md (hunk 2 above)
Comment (Kevan-Y) [reply doc/source/stockholm.md]: https://github.com/slurpcode/slurp/pull/1855/commits/35b083d23969b85d0b9a015ba66ad629a6a4d13a

**Review by jbampton — CHANGES_REQUESTED** [review]
Stockholm a `city` is the capital of Sweden a `country`.

Please update `index.rst`

**Comment by Kevan-Y** [comment]
> Stockholm a `city` is the capital of Sweden a `country`.
> 
> Please update `index.rst`

Oh yeah my bad, put in the wrong section. I will be careful. Thank you

**Review by jbampton — APPROVED** [review]

**Pull request merged**